### PR TITLE
fix: aggregate function throw error when column alias name is set

### PR DESCRIPTION
**Pull request by shenX-2021**
Closes: #9927

### Pull-Request Checklist

- [x] Code is up-to-date with the `master` branch
- [x] `npm run format` to apply prettier formatting
- [ ] `npm run test` passes with this change
- [x] This pull request links relevant issues as `Fixes #0000`
- [ ] There are new or updated unit tests validating the change
- [ ] Documentation has been updated to reflect this change
- [x] The new commits follow conventions explained in [CONTRIBUTING.md](https://github.com/typeorm/typeorm/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/entity-manager/EntityManager.ts [hunk 1]
```diff
@@ -1054,11 +1054,15 @@ export class EntityManager {
...
+        const column = metadata.columns.find(
+            (item) => item.propertyPath === columnName,
+        )
+
         const result = await this.createQueryBuilder(entityClass, metadata.name)
             .setFindOptions({ where })
             .select(
                 `${fnName}(${this.connection.driver.escape(
-                    String(columnName),
+                    column?.databaseName ?? String(columnName),
```
Comment (pleerock) [inline src/entity-manager/EntityManager.ts:1065]: I think it more makes sense to throw an error if column doesn't exist.

**Comment by pleerock** [comment]
Can you please add the test confirming new behavior?

File: src/entity-manager/EntityManager.ts (hunk 1 above)
Comment (shenX-2021) [reply src/entity-manager/EntityManager.ts:1065]: Got it

**Comment by shenX-2021** [comment]
@pleerock Can you merge this PR?

**Comment by pleerock** [comment]
@shenX-2021 You have lint failing.

**Comment by pleerock** [comment]
Not sure why you decided to close it, but in such case close the issue as well.

**Comment by shenX-2021** [comment]
it is something wrong with github, I try opening a new PR

**Pull request closed without merging**